### PR TITLE
"Margin" bug fix when using QuickCSS Popout

### PR DESCRIPTION
**Pull request by SiriusBYT**
Video showing that this indeed works (light mode warning)
https://github.com/replugged-org/replugged/assets/87911136/cd35d9ed-30b7-4fee-a499-a6cedc5c242c

### CODE REVIEW

File: src/renderer/coremods/settings/pages/QuickCSS.css [hunk 1]
```diff
@@ -72,3 +75,19 @@
   width: 100%;
   height: 100%;
 }
+
+/* SB: If you remove this section of the code, the Discord Title bar will show up which is
+annoying since it's black and well with Light Theme it don't make sense bud. */
+[class^="popout-"]:has(#replugged-quickcss-wrapper) [class^="content-"] {
```
Comment (FedeIlLeone) [inline src/renderer/coremods/settings/pages/QuickCSS.css:81]: Remove. All Discord's popouts have the titlebar dark themed, keep it as it is. And the "Stay On Top" button shouldn't be on the titlebar neither.

File: src/renderer/coremods/settings/pages/QuickCSS.css [hunk 2]
```diff
@@ -72,3 +75,19 @@
...
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+/* SB: This is also horrendous, need a class here people.
+It works though. This is part of the real fix. */
+#replugged-quickcss-wrapper > div:last-of-type {
```
Comment (FedeIlLeone) [inline src/renderer/coremods/settings/pages/QuickCSS.css:89]: Not needed.

File: src/renderer/coremods/settings/pages/QuickCSS.css [hunk 3]
```diff
@@ -31,15 +33,16 @@
   min-height: 300px;
 }
 
-#replugged-quickcss-wrapper .cm-scroller {
-  overflow: auto;
+/* #replugged-quickcss-wrapper .cm-scroller {
```
Comment (FedeIlLeone) [inline src/renderer/coremods/settings/pages/QuickCSS.css:36]: Remove

File: src/renderer/coremods/settings/pages/QuickCSS.css [hunk 4]
```diff
@@ -1,5 +1,6 @@
 #replugged-quickcss-wrapper {
   color: var(--text-normal);
+  height: 100%; /* SB: Prevent huge ass margins */
```
Comment (FedeIlLeone) [inline src/renderer/coremods/settings/pages/QuickCSS.css:3]: Delete all these comments.

**Review by FedeIlLeone — CHANGES_REQUESTED** [review]

**Comment by 12944qwerty** [comment]
No longer needed. CSS was sent on [discord](https://discord.com/channels/1000926524452647132/1000955967627874424/1154808243831447562)

**Pull request closed without merging**